src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-cluster.h

```diff
@@ -59,7 +59,7 @@ class PushAvStreamTransportServer : public DefaultServerCluster
 
     CHIP_ERROR Init() { return mLogic.Init(); }
 
-    void Deinit() { mLogic.Shutdown(); }
+    void Shutdown() override { mLogic.Shutdown(); }
```

Comment:
Need to also call DefaultServerCluster::Shutdown for base class processing (in this case that clears the context).